Zadatak3/extern.c

```diff
@@ -1,10 +1,10 @@
 #include <stdio.h>
 
-
-//Manipulacije sa JAVNOM GLOBALNOM promenljivom
-//Definisana je i u file1.c i u fajl2.c
+//Manipulacije sa ZATVORENOM GLOBALNOM promenljivom
+//Definisana je i u file1.c i u fajl2.c bez inicijalizacije
 //fajl1.c je inicijalizuje, a fajl2.c je ispisuje
-
+//output: Globalna promenljiva ima vrednost: 0
```

Comment:
+//output: Globalna promenljiva ima vrednost: 0

Tačno, jer kada imamo 2 zatvorene globalne promenljive tu vrednost smo setovali nekoj drugoj zatvorenoj globalnoj promenljivoj koja igrom slučaja ima isto ime, ali to kompajleru nije bitno.